tools/flux/flux.xml

```diff
@@ -39,7 +39,7 @@ python3 '$__tool_directory__/flux.py'
         </conditional>
     </inputs>
     <outputs>
-        <data name="output" format="png" label="${tool.name} on ${on_string}" from_work_dir="./output.png"/>
+        <data name="output" format="png" label="${tool.name} on ${on_string if $input_type_selector=='file' else 'text prompt'}" from_work_dir="./output.png"/>
```

Comment:
if that a good idea? We don't know how large the prompt will be

Reply:
It doesn't add the prompt it self, it just announces that it is a text prompt.
Before it was empty, if it was not a file.

Reply:
Or should it be a short text about the prompt?

Reply:
No, that is ok. Thanks